Amazon.QLDB.Driver.IntegrationTests/AsyncStatementExecutionTests.cs

```diff
@@ -424,18 +244,7 @@ public async Task ExecuteAsync_InsertMultipleDocuments_DocumentsInserted()
 
```

Comment:
I think it's better if we extract the first 9 lines of this test in a method.

Reply:
Replaced it with `List<IIonValue> parameters = new List<IIonValue> { GetSingleValueIonStruct(IonString1), GetSingleValueIonStruct(IonString2) };`